notifications/src/main/java/org/openedx/notifications/presentation/inbox/NotificationsInboxFragment.kt

```diff
@@ -0,0 +1,483 @@
+                            .size(8.dp)
+                            .background(MaterialTheme.appColors.primaryButtonBackground)
+                    )
+                }
+            }
+            Text(
+                text = TextUtils.dateToRelativeTimeString(LocalContext.current, item.created),
+                style = MaterialTheme.appTypography.bodySmall,
+                color = MaterialTheme.appColors.notificationTimestamp,
+            )
+        }
+    }
+}
+
+@Composable
+fun InboxStateView(
+    modifier: Modifier = Modifier,
+    uiState: InboxUIState,
+    onReloadNotifications: () -> Unit = { },
+) {
+    val iconResId = if (uiState is InboxUIState.Empty) Icons.Outlined.Notifications
+    else Icons.Outlined.SignalWifiStatusbarConnectedNoInternet4
+
+    val titleResId = if (uiState is InboxUIState.Empty) R.string.notifications_no_notifications_yet
+    else coreR.string.core_no_internet_connection
+
+    val descriptionResId =
+        if (uiState is InboxUIState.Empty) R.string.notifications_no_notifications_yet_description
+        else coreR.string.core_no_internet_connection_description
+
+    Column(
+        modifier = modifier.fillMaxSize(),
+        verticalArrangement = Arrangement.Center,
+        horizontalAlignment = Alignment.CenterHorizontally,
+    ) {
+        Box(
+            modifier = Modifier
+                .clip(CircleShape)
+                .size(62.dp)
+                .background(MaterialTheme.appColors.primaryCardCautionBackground)
+                .padding(4.dp),
+        ) {
+            Icon(
+                modifier = Modifier
+                    .size(42.dp)
+                    .align(Alignment.Center),
+                imageVector = iconResId,
+                contentDescription = null,
+                tint = MaterialTheme.appColors.onSurface
+            )
+        }
+        Spacer(modifier = Modifier.height(12.dp))
+
+        Text(
+            text = stringResource(titleResId),
+            style = MaterialTheme.appTypography.titleLarge,
+            color = MaterialTheme.appColors.textPrimary,
+            textAlign = TextAlign.Center,
+        )
+        Spacer(modifier = Modifier.height(12.dp))
+
+        Text(
+            text = stringResource(descriptionResId),
+            style = MaterialTheme.appTypography.bodyLarge,
+            color = MaterialTheme.appColors.textPrimary,
+            textAlign = TextAlign.Center,
+        )
+        Spacer(modifier = Modifier.height(12.dp))
+
+        if (uiState is InboxUIState.Error) {
+            OpenEdXPrimaryButton(
+                modifier = Modifier
+                    .widthIn(Dp.Unspecified, 162.dp),
+                text = stringResource(id = coreR.string.core_reload),
+                textColor = MaterialTheme.appColors.secondaryButtonText,
+                backgroundColor = MaterialTheme.appColors.secondaryButtonBackground,
+                onClick = onReloadNotifications,
+            )
+        }
```

Comment:
According to the [Figma design](https://www.figma.com/design/I9xgrJzdTje5vKgh6DD57C/Mobile-UI-Screens-(2025)?node-id=116-3106&t=EnpEWGB5oFiTm5hh-0), no button has been placed.

Reply:
The button is present in the light mode, Gabriel probably forgot about it in the dark mode.